codespell_lib/_codespell.py

```diff
@@ -43,6 +43,7 @@
     ('clear', 'for unambiguous errors', '', False, None),
     ('rare', 'for rare but valid words', '_rare', None, None),
     ('informal', 'for informal words', '_informal', True, True),
+    ('usage', 'for recommended terms', '_usage', None, None),
```

Comment:
Shouldn't this be True, True (as master will be in Aspell), or does it need to be True, None if allowlist isn't in aspell?

Reply:
`True, None` works. It didn't like `allowlist` and `blocklist` without it.

Reply:
Ah great thanks.